JetMETCorrections/Configuration/python/JetCorrectorsAllAlgos_cff.py

```diff
@@ -520,666 +560,909 @@
 kt6CaloL1FastL2L3L6Corrector = kt6CaloL1L2L3Corrector.clone()
 kt6CaloL1FastL2L3L6Corrector.correctors.append('kt6CaloL6SLBCorrector')
-kt6CaloL1FastL2L3L6CorrectorChain = cms.Sequence(
-    ak7CaloL1L2L3Corrector * kt6CaloL6SLBCorrector * kt6CaloL1FastL2L3L6Corrector
+kt6CaloL1FastL2L3L6CorrectorTask = cms.Task(
+    ak7CaloL1L2L3Corrector, kt6CaloL6SLBCorrector, kt6CaloL1FastL2L3L6Corrector
 )
+kt6CaloL1FastL2L3L6CorrectorChain = cms.Sequence(kt6CaloL1FastL2L3L6CorrectorTask)
 ic5CaloL1FastL2L3L6Corrector = ic5CaloL1L2L3Corrector.clone()
 ic5CaloL1FastL2L3L6Corrector.correctors.append('ic5CaloL6SLBCorrector')
-ic5CaloL1FastL2L3L6CorrectorChain = cms.Sequence(
-    ak7CaloL1L2L3Corrector * ic5CaloL6SLBCorrector * ic5CaloL1FastL2L3L6Corrector
+ic5CaloL1FastL2L3L6CorrectorTask = cms.Task(
+    ak7CaloL1L2L3Corrector, ic5CaloL6SLBCorrector, ic5CaloL1FastL2L3L6Corrector
 )
+ic5CaloL1FastL2L3L6CorrectorChain = cms.Sequence(ic5CaloL1FastL2L3L6CorrectorTask)
 
 ak7PFL1FastL2L3L6Corrector = ak7PFL1FastL2L3Corrector.clone()
 ak7PFL1FastL2L3L6Corrector.correctors.append('ak7PFL6SLBCorrector')
-ak7PFL1FastL2L3L6CorrectorChain = cms.Sequence(
-    ak7CaloL1L2L3Corrector * ak7PFL6SLBCorrector * ak7PFL1FastL2L3L6Corrector
+ak7PFL1FastL2L3L6CorrectorTask = cms.Task(
+    ak7CaloL1L2L3Corrector, ak7PFL6SLBCorrector, ak7PFL1FastL2L3L6Corrector
 )
+ak7PFL1FastL2L3L6CorrectorChain = cms.Sequence(ak7PFL1FastL2L3L6CorrectorTask)
 kt4PFL1FastL2L3L6Corrector = kt4PFL1FastL2L3Corrector.clone()
 kt4PFL1FastL2L3L6Corrector.correctors.append('kt4PFL6SLBCorrector')
-kt4PFL1FastL2L3L6CorrectorChain = cms.Sequence(
-    kt4PFL1FastL2L3Corrector * kt4PFL6SLBCorrector * kt4PFL1FastL2L3L6Corrector
+kt4PFL1FastL2L3L6CorrectorTask = cms.Task(
+    kt4PFL1FastL2L3Corrector, kt4PFL6SLBCorrector, kt4PFL1FastL2L3L6Corrector
 )
+kt4PFL1FastL2L3L6CorrectorChain = cms.Sequence(kt4PFL1FastL2L3L6CorrectorTask)
 kt6PFL1FastL2L3L6Corrector = kt6PFL1FastL2L3Corrector.clone()
 kt6PFL1FastL2L3L6Corrector.correctors.append('kt6PFL6SLBCorrector')
-kt6PFL1FastL2L3L6CorrectorChain = cms.Sequence(
-    kt4PFL1FastL2L3Corrector * kt6PFL6SLBCorrector * kt6PFL1FastL2L3L6Corrector
+kt6PFL1FastL2L3L6CorrectorTask = cms.Task(
+    kt4PFL1FastL2L3Corrector, kt6PFL6SLBCorrector, kt6PFL1FastL2L3L6Corrector
 )
+kt6PFL1FastL2L3L6CorrectorChain = cms.Sequence(kt6PFL1FastL2L3L6CorrectorTask)
 ic5PFL1FastL2L3L6Corrector = ic5PFL1FastL2L3Corrector.clone()
 ic5PFL1FastL2L3L6Corrector.correctors.append('ic5PFL6SLBCorrector')
-ic5PFL1FastL2L3L6CorrectorChain = cms.Sequence(
-    kt4PFL1FastL2L3Corrector * ic5PFL6SLBCorrector * ic5PFL1FastL2L3L6Corrector
+ic5PFL1FastL2L3L6CorrectorTask = cms.Task(
+    kt4PFL1FastL2L3Corrector, ic5PFL6SLBCorrector, ic5PFL1FastL2L3L6Corrector
+)
+ic5PFL1FastL2L3L6CorrectorChain = cms.Sequence(ic5PFL1FastL2L3L6CorrectorTask)
+
+jetCorrectorsAllAlgosTask = cms.Task(
+    jetCorrectorsTask,
+    ak7L1JPTOffsetCorrectorTask,
```

Comment:
the sorting order is somewhat arguable.

I'd group by the first part, preceding LX\*CorrectorTask
and maybe even leave a commented out ```xyzLX\*CorrectorTask``` to be used as a template for other jet algorithms.
We more frequently add an jet algorithm than a new corrector

Also, some separator between subsets (an empty line or ```#LX\*CorrectorTask follow``` or similar) would help with readability.

@schoef @aperloff